source/connect/connect-from-the-command-line.txt

```diff
@@ -38,23 +38,47 @@ Command Line Connection Specification
 -------------------------------------
 
 The command line invocation for |compass-short| has two components, the 
-path to the |compass-short| executable and a connection string. The
+path to the |compass-short| executable and a connection string, you can
+optionally pass in the username and password parameters. The
```

Comment:
I think we decided to reserve the word "parameters" for system settings while you were out. Consider


the path to the |compass-short| executable and a connection string. You can
provide a username and password on the command line or in the <link> configuration file.